src/main/java/be/appwise/core/networking/Networking.kt

```diff
@@ -143,6 +144,11 @@ object Networking {
             return this
         }
 
+        fun registerProxymanService(context: Context,deviceName : String?= null , allowedServices : ArrayList<String> = arrayListOf()) : Builder{
```

Comment:
The spacing between the parameters looks a little off here (Zappware is letting me see things 😅)